projects/angular/src/forms/datepicker/date-container.ts

```diff
@@ -222,4 +274,15 @@ export class ClrDateContainer extends ClrAbstractContainer implements AfterViewI
   private initializeCalendar(): void {
     this.dateNavigationService.initializeCalendar();
   }
+
+  private dateRangeStucturalChecks() {
+    if (this.dateNavigationService.isRangePicker) {
+      if (!this.clrRangeStartEl) {
+        console.error('Error! clr-date-range-container must contain clrRangeStartDate input');
+      }
+      if (!this.clrRangeEndEl) {
+        console.error('Error! clr-date-range-container must contain clrRangeEndDate input');
+      }
+    }
+  }
```

Comment:
This should probably also check for the presence of a regular `ClrDateInput` as well.